flagger/queries.py

```diff
@@ -9,3 +9,12 @@ def find_domains():
         for route in routes:
             domains.extend(route.domain_external_list())
         return domains
+
+
+def find_aliases():
```

Comment:
Do we need to return the `domain_cdns` variable?

Reply:
well that would probably help, yeah